sql/index/pilosa/lookup.go

```diff
@@ -75,11 +80,7 @@ func (l *indexLookup) bitmapQuery() (*pilosa.PQLBitmapQuery, error) {
 		)
 
 		switch il := op.lookup.(type) {
-		case *indexLookup:
-			b, e = il.bitmapQuery()
-		case *ascendLookup:
-			b, e = il.bitmapQuery()
-		case *descendLookup:
+		case pilosaLookup:
```

Comment:
When there is only one case for a switch, is there any difference between use a `switch` and a `if`? or is it just a matter of taste?

Reply:
I just deleted all the other cases and never bothered to make it an if. But yes, I'll make it an if.